### PR TITLE
Feat/sift api

### PR DESCRIPTION
**Pull request by martinhecher**
allows to access the Sift API from within the frontend Sift

If you want to have a look

### CODE REVIEW

File: frontend/src/scripts/view.js [hunk 1]
```diff
@@ -1,27 +1,100 @@
...
+      method: 'POST',
+      path: '/echo',
+      data: repeatMe,
+    });
+
+    return response;
+  }
+
+  // TODO: integrate into sift-sdk-web!
+  sendApiRequest({
```
Comment (jrans) [inline frontend/src/scripts/view.js]: if this is the exposed method to other parts of the app, take apiToken and userAccountId and take it from context instead as not the callers responsibility, but realise this is just first quick demonstration

File: frontend/src/scripts/view.js [hunk 2]
```diff
@@ -1,27 +1,100 @@
...
   }
 
   // for more info: http://docs.redsift.com/docs/client-code-siftview
-  presentView(value) {
-    console.log('botfwk-sift: presentView: ', value);
-    Object.keys(value.data).forEach((k) => {
-      document.getElementById(k).textContent = value.data[k];
-    });
-  };
+  async presentView({ data }) {
```
Comment (jrans) [inline frontend/src/scripts/view.js]: if used as a core sift template may be better to remove async await as may be a barrier to entry for some developers with not as advanced language features, but of course much more beautiful this way!

File: frontend/src/scripts/view.js (hunk 1 above)
Comment (martinhecher) [inline frontend/src/scripts/view.js]: yes, I'll have a look on how to integrate it into `sift-sdk-web`. It's no problem to do the setup with `apiToken` and `userAccountId` for the SiftController (we have full control), but the developer might need to call a setup function in SiftView once in `presentView()`.

File: frontend/src/scripts/view.js (hunk 2 above)
Comment (martinhecher) [inline frontend/src/scripts/view.js]: I'd recommend using the async/await feature with an educational link to it, with an additional link to a branch which provides the same functionality with a promise-only implementation.

Or the other way round, we can discuss this...

**Pull request merged**